_pkgdown.yml

```diff
@@ -1,9 +1,8 @@
 url: https://epiforecasts.io/scoringutils/
 template:
   bootstrap: 5
-  package: preferably
-  params:
-    toggle: manual
+  bslib:
+    font_weight_base : 300
```

Comment:
Might be best to remove because of accessibility issues
https://github.com/epiforecasts/socialmixr/pull/79#discussion_r1274953802

Reply:
I'd rely on the 100/200 recommendation as you mentioned, but don't really know 🤷